src/translations.js

```diff
@@ -53,6 +54,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       my: "Jumlah Bintang",
       sk: "Hviezdy",
       tr: "Toplam Yıldız",
+      uz: "Yulduzchalar",
```

Comment:
First of all, I don't speak Uzbek so I'm not the best person to review this PR. I however quickly checked some obvious things using Google translate. This process is not perfect since Google translate is not a native speaker like you and still makes lots of errors. If you could tag another Uzbek speaking person to quickly review this PR that would be great. 

Google translates states `Yulduzchalar` means `astrix`. Just wanted to double-check why you use this over `
Yigʻilgan umumiy yulduzlar`.